### PR TITLE
Create New User Command and Test

### PR DESCRIPTION
**Pull request by BrandonSPT**

### CODE REVIEW

File: tests/Unit/ArtisanTest.php [hunk 1]
```diff
@@ -37,6 +37,23 @@ public function testFakeUserCanBeCreatedWithNoTenants()
         $this->assertNotEmpty(User::first());
     }
 
+    public function testUserCanBeCreatedWithNoTenants(){
+        // In order to properly test our commands, we need to mock the ask()
+        // and anticipate() methods but leave the rest of the class to behave as normal
+        $this->mockCommand("\MultiTenantLaravel\App\Commands\CreateUser[ask, ask, anticipate]");
+
```
Comment (MyMediaMagnet) [inline tests/Unit/ArtisanTest.php]: I think this should read `CreateUser[asks, anticipates]`

Comment (MyMediaMagnet) [reply tests/Unit/ArtisanTest.php]: Nevermind...singular seems to be how I have it.  You only need to add `ask` once however.

Comment (BrandonSPT) [reply tests/Unit/ArtisanTest.php]: Fixed

File: src/app/Commands/CreateUser.php [hunk 2]
```diff
@@ -45,7 +45,10 @@ public function handle()
         $count = (int) $this->ask('How many would you like to create?');
 
         while ($count > 0) {
+
+
             if (!$this->option('fake')) {
+
```
Comment (MyMediaMagnet) [inline src/app/Commands/CreateUser.php:51]: I know you'll think this is picky, but lets get rid of the extra whitespace here :).

File: src/app/Commands/CreateUser.php [hunk 3]
```diff
@@ -100,6 +103,40 @@ private function createFakeUser()
...
+        $name = (string) $this->ask('Name');
+        $email = (string) $this->ask('E-Mail');
+        $user = config('multi-tenant.user_class')::create([
+            'name' => $name,
+            'email' => $email,
+            'password' => bcrypt('tester'),
+        ]);
+
+
```
Comment (MyMediaMagnet) [inline src/app/Commands/CreateUser.php:116]: extra line break

File: tests/Unit/ArtisanTest.php [hunk 4]
```diff
@@ -37,6 +37,27 @@ public function testFakeUserCanBeCreatedWithNoTenants()
...
+        $this->asks('How many would you like to create?', '1');
+        $this->asks('Name', 'John Doe');
+        $this->asks('E-Mail', 'johndoe@email.com');
+
+        $this->anticipates('Would you like to assign the user to a tenant?', ['Yes', 'No'], 'Yes', 'No');
+
+
+        $this->fireCommand('tenant:create-user');
+
+
```
Comment (MyMediaMagnet) [inline tests/Unit/ArtisanTest.php:54]: line breaks

**Review by MyMediaMagnet — CHANGES_REQUESTED** [review]
Just formatting stuff...all looks good though

**Pull request merged**